test/fixtures/lightscript/arrow-methods/class-malformed-get-fat/actual.js

```diff
@@ -1,3 +0,0 @@
-class X {
-  method() =get> 1
```

Comment:
@rattrayalex a failure case right? I kind of just dropped it because in JS you can't do a fat arrow getter in a class